### PR TITLE
[anari] Added ANARI port.

### PR DESCRIPTION
**Pull request by acdemiralp**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [x] The name of the port matches an existing name for this component on https://repology.org/ if possible, and/or is strongly associated with that component on search engines.
- [x] Optional dependencies are resolved in exactly one way. For example, if the component is built with CMake, all `find_package` calls are REQUIRED, are satisfied by `vcpkg.json`'s declared dependencies, or disabled with [CMAKE_DISABLE_FIND_PACKAGE_Xxx](https://cmake.org/cmake/help/latest/variable/CMAKE_DISABLE_FIND_PACKAGE_PackageName.html)
- [x] The versioning scheme in `vcpkg.json` matches what upstream says.
- [x] The license declaration in `vcpkg.json` matches what upstream says.
- [x] The installed as the "copyright" file matches what upstream says.
- [x] The source code of the component installed comes from an authoritative source.
- [x] The generated "usage text" is accurate. See [adding-usage](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/examples/adding-usage.md) for context.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is in the new port's versions file.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

File: ports/anari/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,31 @@
+vcpkg_minimum_required(VERSION 2022-10-12) # For ${PORT} and ${VERSION}.
+
```
Comment (Adela0814) [inline ports/anari/portfile.cmake:2]: Unnecessary， please remove this,

File: ports/anari/portfile.cmake [hunk 2]
```diff
@@ -0,0 +1,31 @@
+vcpkg_minimum_required(VERSION 2022-10-12) # For ${PORT} and ${VERSION}.
+
+vcpkg_from_github(
+  OUT_SOURCE_PATH SOURCE_PATH
+  REPO KhronosGroup/ANARI-SDK
+  REF v${VERSION}
```
Comment (Adela0814) [inline ports/anari/portfile.cmake:6]:
```suggestion
  REF "v${VERSION}"
```

File: ports/anari/portfile.cmake [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+vcpkg_fixup_pkgconfig()
+vcpkg_copy_pdbs()
+
+file(REMOVE_RECURSE 
+  "${CURRENT_PACKAGES_DIR}/debug/include" 
+  "${CURRENT_PACKAGES_DIR}/debug/share")
+
+vcpkg_install_copyright(
+  FILE_LIST "${SOURCE_PATH}/LICENSE"
+)
```
Comment (Adela0814) [inline ports/anari/portfile.cmake:31]: Please add a new line at the end of the file.

File: ports/anari/vcpkg.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    {
+      "name": "vcpkg-cmake",
+      "host": true
+    },
+    {
+      "name": "vcpkg-cmake-config",
+      "host": true
+    }
+  ]
+}
```
Comment (Adela0814) [inline ports/anari/vcpkg.json:17]: Please add a new line at the end of the file.

**Review by Adela0814 — CHANGES_REQUESTED** [review]

**Comment by Adela0814** [comment]
You should run `vcpkg x-add-version anari` to add the versions file.

**Comment by Adela0814** [comment]
Note: I will be converting your PR to draft status. When you respond, please revert to "ready for review". That way, I can be aware that you've responded since you can't modify the tags.

**Comment by acdemiralp** [comment]
A question: Is it OK for the default implementation library (anari::helide) to depend on an internally downloaded version of Embree? I believe this is an intentional decision by Khronos; to have one "safe" device that works anywhere and does not depend on anything.

If not, it is possible to disable it via `-DBUILD_HELIDE_DEVICE=OFF` in `vcpkg_cmake_configure`.

**Comment by acdemiralp** [comment]
The only remaining error is:
```
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Tools\MSVC\14.37.32822\include\immintrin.h(15): fatal error C1189: #error:  This header is specific to X86, X64, ARM64, and ARM64EC targets
```

This occurs in ARM64 debug mode, release works fine. I am looking for a solution.

**Comment by acdemiralp** [comment]
I think it is good to go.

**Comment by acdemiralp** [comment]
I think its good to go. Checked the installation directories as a last step, everything looks clean.

**Comment by Adela0814** [comment]
> Edit: The error is due to MSVC or Embree. Disabling port on ARM64 Windows until it is fixed.

Please add the failed triplet to the `ci. baseline` instead of the supports field.

**Comment by Adela0814** [comment]
Usage tested pass on `x64-windows`.
No feature needs to test.

**Review by Adela0814 — APPROVED** [review]

**Comment by BillyONeal** [comment]
```
populate-prefix/src/anari_helide_embree-populate-stamp/download-anari_helide_embree-populate.cmake:170 (message):
  Each download failed!

    error: downloading 'https://github.com/embree/embree/archive/refs/tags/v3.13.5.zip' failed
          status_code: 55
          status_string: "Failed sending data to the peer"
          log:
```

Looks like this port is being a bad kitty and trying to recursively download inside the build.

Also it's a bit scary that this isn't using the embree port

**Comment by acdemiralp** [comment]
> ```
> populate-prefix/src/anari_helide_embree-populate-stamp/download-anari_helide_embree-populate.cmake:170 (message):
>   Each download failed!
> 
>     error: downloading 'https://github.com/embree/embree/archive/refs/tags/v3.13.5.zip' failed
>           status_code: 55
>           status_string: "Failed sending data to the peer"
>           log:
> ```
> 
> Looks like this port is being a bad kitty and trying to recursively download inside the build.
> 
> Also it's a bit scary that this isn't using the embree port

I am looking into this now. The fastest way out is to disable the Helide implementation with `-DBUILD_HELIDE_DEVICE=OFF`. The better way is to patch the cmake to use the vcpkg Embree. I'll probably do the latter and open an MR.

**Pull request merged**